v/vault/Dockerfiles/1.5.3-ubi8/README.md

```diff
@@ -7,3 +8,38 @@ Docker run command:
         Server mode:
         docker run --cap-add=IPC_LOCK -e 'VAULT_LOCAL_CONFIG={"backend": {"file": {"path": "/vault/file"}}, "default_lease_ttl": "168h", "max_lease_ttl": "720h"}' ppc64le/vault server
 
+# Deploying Vault on Openshift
+```
+$ oc new-project vault-test
+--> Now using project "vault-test" on server "https://api.sneha-ppc.cp.fyre.ibm.com:6443".
+    You can add applications to this project with the 'new-app' command. For example, try:
+    oc new-app rails-postgresql-example
+    to build a new example application in Ruby. Or use kubectl to deploy a simple Kubernetes application:
+    kubectl create deployment hello-node --image=k8s.gcr.io/serve_hostname
+    
+$ oc adm policy add-scc-to-user anyuid -z default
+--> clusterrole.rbac.authorization.k8s.io/system:openshift:scc:anyuid added: "default"
+
+$ oc new-app ibmcom/vault-ppc64le:1.5.3-ubi8 -e SKIP_CHOWN=true -e SKIP_SETCAP=true
+--> Found container image 6e8b586 (6 hours old) from quay.io for "quay.io/snehakpersistent/vault:latest"
```

Comment:
Please please recheck the image name ?
app is being created with `ibmcom/vault-ppc64le:1.5.3-ubi8` image, but in comment it was mentioned as `quay.io`

Reply:
Updated it to ibmcom/vault-ppc64le:1.5.3-ubi8. I was using the image from quay for testing purpose as the new image is not yet pushed to ibmcom.